### PR TITLE
issue-255

### PR DESCRIPTION
**Pull request by nhomka**
Added support for SimplyWhisked.com

### CODE REVIEW

File: recipe_scrapers/simplywhisked.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, normalize_string, get_yields
```
Comment (bfcarpio) [inline recipe_scrapers/simplywhisked.py:2]: Are you using these at all? If not, please remove them.

**Review by bfcarpio — CHANGES_REQUESTED** [review]
Congrats on your first contribution! This is a nice clean PR, but there are some failures on the test runners. 

https://travis-ci.org/github/hhursev/recipe-scrapers/jobs/742844041

Looks like it's a simple formatting and about those unnecessary imports. Nothing big. Did you setup your dev environment as described in the README?  It would have caught those when you tried to commit.

Clean up the errors and I'll (we'll) merge it in right away! Thanks!

File: recipe_scrapers/simplywhisked.py (hunk 1 above)
Comment (nhomka) [reply recipe_scrapers/simplywhisked.py]: I'm sorry about that, I see what I missed with the pre-commit checks now, I removed the unused imports and fixed the formatting and I think everything should be good now!

Comment (bfcarpio) [reply recipe_scrapers/simplywhisked.py]: Thanks for the update!

**Review by bfcarpio — APPROVED** [review]

**Pull request merged**